Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -10,6 +10,8 @@ public class ParseErrors {
 
   public enum ErrorId {
     g_include_has_errors, // In Reporting.cs
+    p_generic_syntax_error,
+    p_generic_semantic_error,
```

Comment:
Will we want an exception for these ones to avoid having to document them? I get that we need an enum value for these locations in code but they don't seem helpful for users.

Reply:
These are intended to be replaced by more specific errorids. Where that is not possible, probably for the syntax errors automatically generated by CoCo, I agree that we should think about what to say -- there is something useful to say though: there is a syntax error but CoCo doesn't really know the real problem.

Reply:
OK